src/feilmeldinger.tsx

```diff
@@ -22,3 +22,15 @@ export function FeilmeldingManglerFnr() {
         </IntlProvider>
     );
 }
+
+export function IngenTilgangTilBruker() {
+    return (
+        <IntlProvider locale="nb" defaultLocale="nb" messages={tekster}>
+            <div className="container hovedinnhold">
+                <AlertStripeInfoSolid>
+                    <FormattedMessage {...tekster.ingenTilgangBruker} />
```

Comment:
er ikke denne veldig generel ? Vi har flere flere "typer" av dette nå?

Reply:
kan vi kanskje lage en genrel komponent ut av dette for alle alert typene vi har nå

Reply:
1f7c977